<?xml version="1.0" encoding="utf-8"?>
<launch>

    ### ROSBRIDGE ###
    <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch"/>

    ### Semantic_maping ###
    <node name="semantic_mapping" pkg="vimantic" type="node_ViMantic.py" output="screen">
        <param name="topic_result" value="vimantic/SemanticObjects"/>
        <param name="topic_intensity" value="RGBD_4_intensity"/>
        <param name="topic_depth" value="RGBD_4_depth"/>
        <param name="topic_republic" value="vimantic/RGB"/>
        <param name="topic_cnn" value="detectron2_ros/result"/>
        <param name="threshold" value="0.20"/>
        <param name="input_angle" value="90"/>
        <param name="point_cloud" value="false"/>
        <param name="debug" value="true"/>
    </node>

    ### RVIZ ###
    <node name="rviz_player" pkg="rviz" type="rviz"
          args="-d $(find vimantic)/launch/ViMantic_config_rviz.rviz"/>

    ### simulation ###
    <node pkg="rosbag" type="play" name="player" output="log"
          args="--clock /home/omega100/Descargas/dataset/rosbagfile/anto-s1_fullhouse1_localization.bag"/>
    <include file="$(find missions_pkg)/launch/simbot/simbot_map_server.launch">
        <arg name="map_file" value="$(find missions_pkg)/maps/anto-s1.yaml"/>
    </include>

</launch>
